app/models/spacy.py

```diff
@@ -0,0 +1,13 @@
+from app.sen_py import SenPy
+import spacy
+
+class SpaCy(SenPy):
```

Comment:
It looks like you have two `SpaCy` models in this PR? Copy/paste error?

Reply:
the import statement here gets the actual spaCy library that our best model runs on. Do i import it elsewhere too? I'll check later